GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -159,5 +159,11 @@ public override bool IsGitStatusCacheSupported()
         {
             return new MacFileBasedLock(fileSystem, tracer, lockPath);
         }
+
+        public override bool KillProcessTree(int processId)
+        {
+            ProcessResult result = ProcessHelper.Run("kill", $"{processId}");

```

Comment:
This does not kill the tree, you wanna take a look at [`pkill`](https://unix.stackexchange.com/questions/124127/kill-all-descendant-processes).

Reply:
Thanks updated!